Source/Serilog.Enrichers.Span/ActivityEnricher.cs

```diff
@@ -33,15 +33,25 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
                 AddSpanId(logEvent, activity);
                 AddTraceId(logEvent, activity);
                 AddParentId(logEvent, activity);
+                AddTags(logEvent, activity);
 #else
                 logEvent.AddPropertyIfAbsent(new LogEventProperty(SpanId, new ScalarValue(activity.GetSpanId())));
                 logEvent.AddPropertyIfAbsent(new LogEventProperty(TraceId, new ScalarValue(activity.GetTraceId())));
                 logEvent.AddPropertyIfAbsent(new LogEventProperty(ParentId, new ScalarValue(activity.GetParentId())));
 #endif
             }
         }
+
 #if NET5_0_OR_GREATER
 
+        private static void AddTags(LogEvent logEvent, Activity activity)
+        {
+            foreach (var item in activity.Tags)
+            {
+                logEvent.AddPropertyIfAbsent(new LogEventProperty(item.Key, new ScalarValue(item.Value)));
```

Comment:
We avoid extra allocations of `LogEventProperty` in the methods below, we should be able to do the same here.

Reply:
I have a question.

The SpanId is immutable in the activity. But Tags is different. 
All logging should be check a new members of Tags.
Correct??

Reply:
Good point, I guess we can't cache the `LogEventProperty` then.